tests/system/action/user/test_set_password_self.py

```diff
@@ -10,7 +7,9 @@ def test_update_correct_permission(self) -> None:
         self.user_id = self.create_user("test", group_ids=[1])
```

Comment:
The tests name `test_update_correct_permission` is not fitting.

Reply:
Renamed.